### DISCUSSION TITLE
Load control possible?

### DISCUSSION THREAD

**Discussion by herrlanders** [cyrils/renogy-bt#75] [General]
Howdy,

I'm cursed with a Renogy Rover that is at a remote property 600 kilometers away that does not re-establish the load if there is a long spell of cloudy weather that triggers the low voltage cutoff.

I have a raspberry pi I can access remotely that can grab all the stats from the rover, that's great but I can't for the life of me find a way to control the load on the Rover so I don't have to travel 600KM to push the button physically. I note in your RoverClient.py there is a write function and a load state value, is it possible this can be used to trigger the load remotely?

Thanks in advance

**Comment by cyrils** [comment]
Yes, this is totally possible, check issue https://github.com/cyrils/renogy-bt/issues/69
